docs/testing-guidelines/getting-code-coverage.md

```diff
@@ -0,0 +1,127 @@
+Getting Code Coverage Analysis for PowerShell
+===
+
+**Note: Code coverage is currently only supported on Windows, since we use OpenCover**
+
+The PowerShell code base is configured to build with code coverage support using [OpenCover](https://github.com/OpenCover/opencover). You can see the testing coverage of the current [`master`](https://github.com/PowerShell/PowerShell) branch build at any time at [codecov.io](https://codecov.io/gh/PowerShell/PowerShell).
+
+To run test coverage analysis of PowerShell on your own branch/machine
+however, you will need to take the following steps (and be aware that running
+the tests under code coverage analysis can take as long as 8 hours).
+
+Running tests with code coverage analysis
+---
+
+**First**: Open PowerShell in an **Administrator** session. OpenCover
+needs Administrator privileges to work.
+
+Now, in PowerShell:
+```powershell
+# Go to your PowerShell build directory root
+PS> Set-Location "C:\Path\to\powershell\build\dir"
+
+# Import the PowerShell build module
+PS> Import-Module .\build.psm1
+
+# Build PowerShell. You may need to add other flags here like
+# -ResGen, -Restore or -PSModuleRestore
+PS> Start-PSBuild -Configuration CodeCoverage
+
+# Save the built executable's location for later use
+PS> $psDir = Split-Path -Parent (Get-PSOutput)
+
+# Now ensure Pester is installed
+PS> Restore-PSPester
+
+# Import the OpenCover module
+PS> Import-Module $PWD\test\tools\OpenCover
+
+# Install OpenCover to a temporary directory
+PS> Install-OpenCover -TargetDirectory $env:TEMP -Force
```

Comment:
I thought we updated this cmdlet so it has a default `TargetDirectory` so it doesn't need to be specified?

Reply:
The original blog post gives an argument (except with no parameter, and `Install-OpenCover`'s first positional argument is actually the URL from which to download OpenCover). I'm happy with using a default, but in that case I think we should write some documentation of build function parameters so that `Get-Help` has some description of what the parameters do.

Reply:
Looking at the file, the current default `TargetDirectory` is `$HOME`

Reply:
If the defaults work, I would recommend not adding additional parameters as it makes it more confusing to new users